### DISCUSSION TITLE
SSR only without Hybrid Rendering - classic html

### DISCUSSION THREAD

**Discussion by dusta** [nuxt/nuxt#28118] [Questions]
I need to test server load and I need to have only SSR (but no by generate static file) I need run app 100% SSR without Hybrid Rendering. Just every click reload classic browser and load html without post load json/ajax.

Is there any option to do this in config without rewriting code?

**Comment by huang-julien** [comment]
hello you can try enabling `experimental.noScripts` in your config !
